File: app/src/main/java/com/seankim/zomatosearchex/Model/RspPopularityModel.java
package com.seankim.zomatosearchex.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RspPopularityModel {

    @SerializedName("popularity")
    @Expose
    Double popularity;

    @SerializedName("nightlife_index")
    @Expose
    Double nightlife_index;

    @SerializedName("nearby_res")
    @Expose
    String[] nearby_res;

    @SerializedName("top_cuisines")
    @Expose
    String[] top_cuisines;

    @SerializedName("popularity_res")
    @Expose
    Double popularity_res;

    @SerializedName("nightlife_res")
    @Expose
    Double nightlife_res;

    @SerializedName("subzone")
    @Expose
    String subzone;

    @SerializedName("subzone_id")
    @Expose
    Integer subzone_id;

    @SerializedName("city")
    @Expose
    String city;
}

File: app/src/main/java/com/seankim/zomatosearchex/Model/RspPopularityModel.kt
package com.seankim.zomatosearchex.Model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RspPopularityModel {

    @SerializedName("popularity")
    @Expose
    internal var popularity: Double? = null

    @SerializedName("nightlife_index")
    @Expose
    internal var nightlife_index: Double? = null

    @SerializedName("nearby_res")
    @Expose
    internal var nearby_res: Array<String>? = null

    @SerializedName("top_cuisines")
    @Expose
    internal var top_cuisines: Array<String>? = null

    @SerializedName("popularity_res")
    @Expose
    internal var popularity_res: Double? = null

    @SerializedName("nightlife_res")
    @Expose
    internal var nightlife_res: Double? = null

    @SerializedName("subzone")
    @Expose
    internal var subzone: String? = null

    @SerializedName("subzone_id")
    @Expose
    internal var subzone_id: Int? = null

    @SerializedName("city")
    @Expose
    internal var city: String? = null
}
